source/sessions/driver-sessions.rst

```diff
@@ -801,15 +802,15 @@ corresponding ``ServerSession``. However, starting a server session might requir
 round trip to the server (which can be avoided by generating the session ID
 locally) and ending a session requires a separate round trip to the server.
 Drivers can operate more efficiently and put less load on the server if they
-cache ``ServerSession`` instances for reuse. To this end drivers SHOULD
+cache ``ServerSession`` instances for reuse. To this end drivers MUST
 implement a server session pool containing ``ServerSession`` instances
 available for reuse. A ``ServerSession`` pool MUST belong to a ``MongoClient``
 instance and have the same lifetime as the ``MongoClient`` instance.
 
-If a driver has a server session pool, then when a new ``ClientSession`` is started
-it MUST attempt to acquire a server session from the server session pool. See
-the algorithm below for the steps to follow when attempting to acquire a
-``ServerSession`` from the server session pool.
+When a new implicit ``ClientSession`` is started it MUST NOT attempt to acquire a server
```

Comment:
why did you change `MUST` to `MUST NOT`? if we can't acquire a server session from the pool, then where would we get the sessions?

Reply:
I think the sentence cuts short, I added "immediately" the change is that specifically an implicit session needs to delay its acquiring until connection checkout but that's described in the "See the algorithm below" bit. LMK if this clears things up enough